README.md

```diff
@@ -137,7 +138,7 @@ format specifications to be used.
 - [iBored](https://apps.tempel.org/iBored/) - Cross-platform, sector based hex editor
 - [Synalyze It!](https://www.synalysis.net) - Hex editor with templates for binary analysis
 - [wxHex Editor](http://www.wxhexeditor.org) - Cross-platform editor with file comparison
-- [hexalepis](https://github.com/sealmove/hexalepis) - Win/Unix gui+terminal, [tweak](https://www.chiark.greenend.org.uk/~sgtatham/tweak/btree.html) engine, .ksy visualization
+- [hexalepis](https://github.com/sealmove/hexalepis) - Implements [tweak](https://www.chiark.greenend.org.uk/~sgtatham/tweak/btree.html) engine (WIP)
```

Comment:
You should write some description of the _hexalepis_ project here. The fact that it implements some efficient engine _tweak_ is not really relevant for the purpose of this awesome-kaitai list (if you consider it noteworthy, you can mention it, but it's not the main point). Instead, it should be like "hexalepis - Efficient hex editor in the terminal".

Reply:
Okay, fair enough, it can be removed from the list.